tests/app/test_model.py

```diff
@@ -181,9 +181,9 @@ def test_notification_subject_fills_in_placeholders_for_email(sample_email_templ
     assert noti.subject == 'hello'
 
 
-def test_notification_subject_fills_in_placeholders_for_letter(sample_letter_template):
-    noti = create_notification(sample_letter_template, personalisation={'name': 'hello'})
-    noti.template.subject = '((name))'
+def test_notification_subject_fills_in_placeholders_for_letter(sample_service):
+    template = create_template(service=sample_service, template_type='email', subject='((name))')
```

Comment:
`template_type` here should be 'letter'

Reply:
Ah good point - or I could parametrise that and test both email and letter